update_scripts/pyiron_base_0.3_to_0.4.py

```diff
@@ -0,0 +1,15 @@
+"""
+pyiron_base<=0.3.9 has a bug that writes all arrays with dtype=object even
+numeric ones.  As a fix pyiron_base=0.3.10 introduces a conversion when reading
```

Comment:
Actually, the real fix on the reading side is introduced here and will be in release 0.4.0
```suggestion
numeric ones.  As a fix pyiron_base=0.4.0 introduces a conversion when reading
```